components/bear-market-airdrop/Blocks/Hero/Unboxed.tsx

```diff
@@ -0,0 +1,295 @@
+/* eslint-disable no-restricted-imports */
+
+/* eslint-disable react/forbid-dom-props */
+import { gradientMapping } from "../Prizes/Prize";
+import {
+  Box,
+  Button,
+  ButtonGroup,
+  Card,
+  CardBody,
+  Flex,
+  Heading,
+  Image,
+  Text,
+} from "@chakra-ui/react";
+import { Polygon } from "@thirdweb-dev/chains";
+import { ThirdwebNftMedia, useAddress } from "@thirdweb-dev/react";
+import { NFT, NFTMetadata, TransactionResult } from "@thirdweb-dev/sdk";
+import { FC } from "react";
+import { BsEyeFill } from "react-icons/bs";
+
+interface UnboxedProps {
+  reward: NFT | undefined;
+  tx: TransactionResult | null;
+}
+
+type Metadata = NFTMetadata & {
+  attributes: {
+    trait_type: string;
+    value: string;
+  }[];
+};
+
+const deliveryMethodMapping: {
+  [key: string]: string;
+} = {
+  "Bear Market Builder NFT": "pack",
+  "thirdweb pro early access NFT": "pack",
+  "5 Testnet MATIC": "airdrop",
+  "1 MATIC": "pack",
+  "10 MATIC": "pack",
+  "1k AWS Credits": "email",
+  "Lexica AI Art Pass": "email",
+  "Quicknode Credits: $300": "email",
+  "Quicknode Credits: $900": "email",
+  "Quicknode Credits: $5000": "email",
+  "Quicknode Credits: $25000": "email",
+  "Club IRL NFT": "In Pack",
+  "Consensus 2023 Two-Day Pass": "email",
+  "Checks NFT (VV Edition)": "airdrop",
+};
+
+const Unboxed: FC<UnboxedProps> = ({ reward, tx }) => {
+  const nft = reward?.metadata as unknown as Metadata;
+  const rarity = nft?.attributes[0]?.value;
+  const chain = process.env.NEXT_PUBLIC_CHAIN;
+  const editionAddress = process.env.NEXT_PUBLIC_EDITION_ADDRESS;
+  const address = useAddress();
+  const image = nft.image?.replaceAll(/ /g, "%20");
+  const twitterUrl = `https://twitter.com/intent/tweet?text=I%20just%20unboxed%20a%20${rarity}%20prize%20on%20%40thirdweb%20and%20it%20is%20a%20${nft?.name}%20${image}`;
+  const facebookUrl = `https://www.facebook.com/sharer/sharer.php?u=${image}`;
+  const instagramUrl = `https://www.instagram.com/sharer/sharer.php?u=${image}`;
+
+  if (!nft) {
+    return null;
+  }
+  const method = deliveryMethodMapping[nft?.name ?? ""] || "pack";
+
+  return (
+    <Flex direction="column" w="full" pb={16} alignItems="center" color="white">
+      <Box
+        fontSize={{
+          base: "2.5rem",
+          xl: "3.5rem",
+        }}
+        position="relative"
+        mt="74px"
+        mb={5}
+      >
+        <Image
```

Comment:
all `<Image>` components should be using `<ChakraNextImage>` (from `tw-components`) here and "require()" the src file directly so we get image optimization out of the box (u can look for other places where we do this)